nautobot/docs/plugins/development.md

```diff
@@ -316,7 +316,54 @@ class AnimalType(DjangoObjectType):
 graphql_types = [AnimalType]
 ```
 
-## Web UI Views
+#### Using GraphQL ORM Utility
```

Comment:
Just a note that this change will conflict with the changes being made in #447

Reply:
#447 is merged; please update your branch and address merge conflicts. Thank you!

Reply:
This subsection should be moved down to the "GraphQL Types" section below (which might need a rename?) and the sections above should be removed as they're redundant now.

Reply:
I changed the name to `Extending GraphQL`. Does that make sense?

Reply:
Perhaps "Integrating with GraphQL"? My thinking is that using the GraphQL ORM utility functions is *using* GraphQL, not *extending* it.